client/vite.config.playground.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'vite';
+import solid from 'vite-plugin-solid';
+// @ts-ignore
+import path from 'path';
+
+export default defineConfig({
+  root: 'src/_playground',
+  build: {
+    outDir: path.join(__dirname, 'src', '_playground', 'dist'),
+    rollupOptions: {
+      input: {
+        app: path.join(__dirname, 'src', '_playground', 'index.html'),
+      },
+    },
+  },
+  plugins: [solid()],
+  resolve: {
+    dedupe: ['react', 'react-dom'],
```

Comment:
We don't use react, so why do we need that?

Reply:
Just reused most of the configuration from the existing config. Good question I am gonna check it